.eslintrc

```diff
@@ -44,6 +44,7 @@
     "react/prefer-stateless-function": 0,
     "react/no-array-index-key": 0,
     "react/no-unused-prop-types": 0,
+    "react/require-default-props": 0,
```

Comment:
I don't see any other way of doing it - as I mentioned yesterday, people suggest that we can ignore it here because flow is actualy taking care about it.